### ISSUE TITLE
Investigate Open Source community management strategy

### ISSUE THREAD

**Issue by achekulaev** [docksal/docksal#591]
Currently Gitter is the only community support channel.

We need to investigate on the future of the Docksal community and tools to be used.

**Comment by sean-e-dietrich** [comment]
My suggestion is to use something like [LaunchPass](https://launchpass.com/) or Slackin as the front end. LaunchPass is not something we have to worry about and is a service that possibly opens up more doors for us in the future. It is also free to start off with. I'm voting we go towards LaunchPass.

My main reason behind the Slack community is it would promote more user engagement. We can do topic specific channels as well as communicate quicker with the users. It would allow for us to get users to test new features. It has endless possibilities. I'm not saying we should get rid of Gitter. I believe we should just go more in the slack direction.

**Comment by lmakarov** [comment]
[Slack vs Gitter](http://qr.ae/TUpN9t) on Quora:

> Gitter is user-centric, Slack is company-centric.

Let's also not forget about forum options

- https://forums.docker.com/ (Discourse)
- https://discuss.circleci.com/ (Discourse)

**Comment by achekulaev** [comment]
I thought over the forum option since we discussed it and I must say I was almost never been able to find reasonable help in software forums. 
Also users will expect us to read it. 
Which we will not be doing, believe me. I moderated forums. We don't have time for this.

So I rather feel like "nay" that "yay" about forums.

**Comment by achekulaev** [comment]
> Gitter is user-centric, Slack is company-centric.

I see this as a good argument in favor of Slack, since we want to solidify the Docksal community and make it stand out a community rather than techie personal help IRC channel.

**Comment by sean-e-dietrich** [comment]
Exactly, I think we need to start making Docksal more of a product and brining the community together so Slack makes the most sense to move towards.

**Comment by achekulaev** [comment]
@lmakarov we need to define what we want. Long term solution with possibility of future adaptations of premium features, or a cost efficient solution that will work right here and right now. 
Spoiler: we can't have both.

**Comment by lpeabody** [comment]
+1 for Discord.

**Comment by lmakarov** [comment]
A few updates, as this topic has recently popped up internally.

We considered https://spectrum.chat/ at some point as a much better replacement for Gitter. However, Github bought them, pivoted the focus to Github Discussions and will soon [sunset spectrum.chat](https://spectrum.chat/spectrum/general/join-us-on-our-new-journey~e4ca0386-f15c-4ba8-8184-21cf5fa39cf5).

Github Discussions is the new kid on the block and is apparently here to stay.
The feature is enabled and available at https://github.com/docksal/docksal/discussions

While Discussions provide a much better knowledge archive (think Stack Overflow), they do not support real-time communications (chat). For that last piece, we can go with Discord as a favored option (setup in progress by @lpeabody)

We also need a set or rules to route communications between Discussions (forum) and Discord (realtime chat).
